debian/control

```diff
@@ -69,13 +69,12 @@ Build-Depends: automake,
                librabbitmq-dev,
                librdkafka-dev,
                luarocks,
-               libyaml-cpp-dev <pkg.ceph.crimson>,
+	        libyaml-cpp-dev (>= 0.6) <pkg.ceph.crimson> <pkg.ceph.jaeger>,
```

Comment:
please use spaces. any reason to move it here?

Reply:
> yaml-cpp will be build from source if enabled.

how is this related this commit? if yes, how is this implemented?

Reply:
in systems like bionic, yaml-cpp >= 0.6 is still not available, for them we build yaml-cpp from the source.
For all other cases we can use >= 0.6, since for our ci/cd we use focal, using pkg version would be more convenient.

why 0.6? >> from 0.6 there is no need for boost dep, if we try to install the previous yaml version, it will conflict with our installed boost